apps/wellknown/views/application.py

```diff
@@ -13,6 +16,44 @@ class ApplicationListPagination(PageNumberPagination):
     max_page_size = 10000
 
 
+class LabelSlugsValidator:
+    def __call__(self, value):
+        if value:
+            slugs = [slug.strip() for slug in value.split(',')]
+            for slug in slugs:
+                if not ApplicationLabel.objects.exclude(slug__in=settings.APP_LIST_EXCLUDE).filter(slug=slug).exists():
```

Comment:
maybe this validator can be separate, checking that there is no overlap between the given list and the `APP_LIST_EXCLUDE` list.